sdk/log/record.go

```diff
@@ -192,11 +194,15 @@ func (r *Record) AddAttributes(attrs ...log.KeyValue) {
 	if n == 0 {
 		// Avoid the more complex duplicate map lookups below.
 		var drop int
-		attrs, drop = dedup(attrs)
-		r.setDropped(drop)
```

Comment:
Do I see correctly that there was a bug and `logAttrDropped` was called even if nothing was actually dropped (and you fixed it as well)?